bot.js

```diff
@@ -1,197 +1,91 @@
 require("dotenv").config();
 
 let TelegramBot = require("node-telegram-bot-api");
+let game = require("./game.js");
 
 if (!process.env.TELEGRAM_TOKEN) {
   throw new Error("TELEGRAM_TOKEN env variable is missing");
 }
 
 const bot = new TelegramBot(process.env.TELEGRAM_TOKEN, { polling: true });
-let gameStart = false;
-let spentCities = new Set();
-let lastLetter = "";
 
-bot.on("polling_error", m => console.log(m));
+let session = [];
```

Comment:
The array is not really the best way to store session. 
Because when you need to get a session for specific user you need to find this session in array. 

Will be better to have an object
```js
let sessions = { 
   // 124125 - chat.id
   124125: { spentCities: new Set() } 
}
```

Reply:
Ok. I will try to implement this

Reply:
```suggestion
let sessions = {
  makeSession(chatID) {
    this[chatID] = { spentCities: new Set() };
  },
  deleteSession(chatID) {
    delete this[chatID];
  }
};
```
fixed it